src/validations/test/sar.xspec

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<x:description
+    schematron="../rules/sar.sch"
+    xmlns:doc="https://fedramp.gov/oscal/fedramp-automation-documentation"
+    xmlns:sch="http://purl.oclc.org/dsdl/schematron"
+    xmlns:sqf="http://www.schematron-quickfix.com/validator/process"
```

Comment:
`http://www.schematron-quickfix.com/validator/process` is a Schematron QuickFix (SQF)-related namespace. It is used to declare Schematron quick fix constructs in a **Schematron document**. This is an XSpec document. Why is SQF declared here?